/axis_camera_driver:
  ros__parameters:
    ip_address: '172.16.0.9'
    username: 'root'
    password: '1234'
    pan0: 148.92
    publish_split: True
    timer_period: 0.1
